sepolia/config.yaml

```diff
@@ -0,0 +1,76 @@
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
+CONFIG_NAME: 'sepolia'
+
+# Genesis
+# ---------------------------------------------------------------
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 1300
```

Comment:
```suggestion
MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: len(genesis_state.validators)
```

why not this?

Reply:
We can increase it, but since the min value is below the len, it shouldn't matter